### ISSUE TITLE
cy.viewport() iPad preset orientation's landscape and portrait are inversed

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#100]
The following command sets the screen size to `768` height and `1024` width. An ipad2 in portrait should have `1024` height and `768` width.

```
cy.viewport("ipad-2", "portrait")
```

The following command sets the screen size to `1024` height and `768` width. An ipad2 in landscape should have `768` height and `1024` width.

```
cy.viewport("ipad-2", "landscape")
```

**Comment by jennifer-mann** [comment]
This is fixed in the upcoming `0.14.0` release.

**Comment by brian-mann** [comment]
Fixed in [`0.14.0`](https://github.com/cypress-io/cypress/wiki/Changelog#0140-03082016).
